plugins/modules/jira.py

```diff
@@ -59,6 +59,18 @@
       - The personal access token to log-in with.
       - Mutually exclusive with O(username) and O(password).
     version_added: 4.2.0
+  client_cert:
+    type: path
+    description:
+      - Client certificate if required.
+      - Can coexist with O(username) and O(password). Not mutually exclusive.
```

Comment:
Can it also co-exist with `token`? In general I would not mention that something is not mutually exclusive, only if there's a reason to assume it is. (So far the reason was that you explicitly wrote that it was mutually exclusive.)
```suggestion
```